style.css

```diff
@@ -1 +1,199 @@
 /*  Add your styling here */
+@import url('https://fonts.googleapis.com/css?family=Courgette|Lato:400,700,900|Raleway|Roboto:400,700&display=swap');
+* {
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Courgette;
+    margin:0;
+    color: #50210d;
+    padding: 1px;
+    background: #F5C7DE;
+    height: auto;
+  }
+  
+  p{
+      padding: 0px 5px;
+  }
+
+  /* Header */
+  .header {
+    padding: 10px;
+    text-align: center;
+    
+  }
+  
+  .header h1 {
+    font-size: 2rem;
+    color:#825947;
+  }
+
+  h2 {
+      color:#b81c4c
+  }
+
+  main{
+      max-width: 900px;
+      margin:auto;
+      box-shadow: 30px 0px 40px rgba(0,0,0,0.4), -30px, 0px, 40px rgba(0,0,0,0.4); /*not working*/
+  }
+  /* Style the top navigation bar */
+  .topnav {
+    overflow: hidden;
+    margin-top: 10;
+    background-color: #eb9de0;
+  }
+  
+  /* Style the topnav links */
+  .topnav a {
+    display: block;
+    background-color: rgb(233, 159, 196);
+    color: #850a2f;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    float: none;
+    width: 100%;
+    
+  }
+  
+  /* Change color on hover */
+  .topnav a:hover {
+    background-color: #b81c4c;
+    color: white;
+  }
+  
+  /* Create two equal columns that floats next to each other */
```

Comment:
i'm not sure if this comment matches what the styling describes here - if you do put comments make sure it's consistent with the output to avoid confusion when you look back on the code later; if you were to do what the comment describes, you would look at using flexbox on the parent class to try and implement this - which is what your `row` class does 😄